src/renderer/src/application/scene_shape.h

```diff
@@ -4,26 +4,16 @@
 #include <glm/gtc/quaternion.hpp>
 #include <memory>
 
+#include "renderer/scene_object.h"
 #include "shapes.h"
 #include "renderer/vertexarray.h"
 
-class SceneShape {
+class SceneShape : public SceneObject {
```

Comment:
do we need word `scene` in SceneShape ? isn't ot obvious from the context? Or there is conflicts with Shape from core?